Solutions/Corvus.Storage.Specs/Corvus/Storage/Sql/SqlDatabaseConfigurations.feature

```diff
@@ -0,0 +1,19 @@
+﻿Feature: SqlDatabaseConfigurations
+    As the person responsible for deploying and configuring an application that use Azure SQL or SQL Server
```

Comment:
Tiny trivial typo - `use => uses`

Reply:
...that appeared in a vast number of files. Fixed!